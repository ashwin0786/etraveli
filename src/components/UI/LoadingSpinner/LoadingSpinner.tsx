import React from 'react';
import './LoadingSpinner.scss';

const LoadingSpinner: React.FC = () => {
    return <div className="spinner-wrapper"><div className="spinner"></div></div>;
};
export default LoadingSpinner;
