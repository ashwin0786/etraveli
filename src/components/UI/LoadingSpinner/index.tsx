import React from 'react';
import './LoadingSpinner.scss';

const LoadingSpinner: React.FC = () => {
    return <div className="spinner"></div>;
};
export default LoadingSpinner;
