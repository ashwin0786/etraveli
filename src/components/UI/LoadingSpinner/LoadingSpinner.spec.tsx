import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import LoadingSpinner from '.';

describe('LoadingSpinner component', () => {
    it('render spinner', () => {
        const { container } = render(<LoadingSpinner />);
        expect(container.firstChild).toMatchSnapshot();
    });
});
