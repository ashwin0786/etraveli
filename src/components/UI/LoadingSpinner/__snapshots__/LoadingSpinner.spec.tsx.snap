// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner component render spinner 1`] = `
<div
  class="spinner-wrapper"
>
  <div
    class="spinner"
  />
</div>
`;
